test/conformance/cf_header.md

```diff
@@ -30,6 +31,7 @@ documentation for that test.
 ### **Output:**
 ## [Kubelet-OutputToLogs](https://github.com/kubernetes/kubernetes/blob/release-1.9/test/e2e_node/kubelet_test.go#L42)
 
+### Release v1.9
```

Comment:
Why are some v1.9 templatized but others not? Can we get notes on the PR or an issue to explain the problem you are solving?